### PR TITLE
Adding superagent

### PR DESCRIPTION
**Pull request by SirJackovich**

### CODE REVIEW

File: fs-globals.html [hunk 1]
```diff
@@ -1 +1,3 @@
-<script src='fs-globals.js'></script>
\ No newline at end of file
+<script src='fs-globals.js'></script>
+<script src='libs/superagent-defaults.js'></script>
+<script src='libs/superagent.js'></script>
```
Comment (redbugz) [inline fs-globals.html]: I don't think these should go here, unless this becomes the full library, but I would prefer it remain the core, and we add an additional fs-globals-all or something, or we just let consumers choose the add-ons they need

File: fs-superagent.html [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<script src='libs/superagent-defaults.js'></script>
+<script src='libs/superagent.js'></script>
```
Comment (redbugz) [inline fs-superagent.html]: 👍

File: .travis.yml [hunk 3]
```diff
@@ -1,49 +1,21 @@
...
-# Branch whitelist
-branches:
-  only:
-    - master
-dist: trusty
+  - echo -e "machine github.com\n  login $CI_USER_TOKEN" >> ~/.netrc
+  - git config --global url.https://github.com/.insteadOf git://github.com/
+  - export CHROME_BIN=chromium-browser
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
```
Comment (redbugz) [inline .travis.yml]: is there a reason we need all of this changed?

Comment (SirJackovich) [reply .travis.yml]: no the test have never ran in travis, I changed the file to get them to work.

File: fs-superagent.html (hunk 2 above)
Comment (gechols) [reply fs-superagent.html]: So the 'opt-in' is to use fs-superagent instead of fs-globals, right? I like it.

Comment (SirJackovich) [reply fs-superagent.html]: I think the opt in is in addition to fs-globals.

Comment (gechols) [reply fs-superagent.html]: OK - works for me.

**Pull request merged**